python/sidecar/search.py

```diff
@@ -0,0 +1,256 @@
+"""
+Author: Shaurya
+This script performs a search on Semantic Scholar and retrieves information about a set of papers.
+
+Instructions:
+The script takes two command line arguments:
+--query: The term you want to search on Semantic Scholar, written in quotation marks (e.g., "brain computer interface").
+--limit: Optional, the maximum number of papers to retrieve. Default is 10.
+
+Run the script like so from your terminal:
+    python search.py --query "computational linguistics" --limit 10
+"""
+import argparse
+from semanticscholar import SemanticScholar
+
+
+class Search:
+    # Initialize the SemanticScholar API client and stopwords set
+    def __init__(self):
+        self.s2 = SemanticScholar()
+        self.stopwords = set(
+            [
```

Comment:
@shauryr can you move these stopwords to a different file?

Reply:
Agree - we can just have a `stopwords.py` in the sidecar directory

Reply:
`stopwords.py` has been added - https://github.com/refstudio/refstudio/pull/250/commits/d089fa182f698b0bd254382c5eece50255ba5074